<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_main">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="Btn_go_userin">
          <property name="text">
           <string>회원등록</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_go_clnt_info">
          <property name="text">
           <string>회원정보 확인 / 수정</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_go_exe_check">
          <property name="text">
           <string>운동루틴 추가 / 확인</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_go_close">
          <property name="text">
           <string>마감</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_user_info">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="userin_save_Btn">
          <property name="text">
           <string>저장</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="deadline_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/image/closing3.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_exe_check">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <widget class="QFrame" name="frame_list">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item>
            <widget class="QPushButton" name="save_Btn">
             <property name="text">
              <string>저장</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2" colspan="2">
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>431</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="grid_name">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>433</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_namewidget"/>
          </widget>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="tr_Btn1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tr_Btn2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tr_Btn3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tr_Btn4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tr_Btn5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tr_Btn6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_pt_day">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_7" rowstretch="3,0,0,0,0,0,0,0,0,0,0,0">
             <item row="2" column="1">
              <widget class="QLineEdit" name="ph_lineEdit">
               <property name="text">
                <string>01012345678</string>
               </property>
               <property name="maxLength">
                <number>11</number>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>담당자</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>이름</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QSpinBox" name="pt_spinbox"/>
             </item>
             <item row="11" column="0" colspan="2">
              <widget class="QPushButton" name="userin_save_Btn_2">
               <property name="text">
                <string>저장하기</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>생년월일</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">image: url(:/image/addmember.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="name_lineEdit">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="bt_lineEdit">
               <property name="text">
                <string>YYYYMMDD</string>
               </property>
               <property name="maxLength">
                <number>8</number>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QComboBox" name="tr_combobox"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>PT횟수</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>휴대폰번호</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Btn_go_main">
      <property name="text">
       <string>메인으로</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
